why-apollo-advantages-disadvantages-alternatives.md

```diff
@@ -23,25 +23,25 @@ summary = "It is not always simple to pick the right solution for a problem. In
 
-Before we dive into the advantages and disadvantages of Apollo, let's answer the following question first: What is Apollo? It seems to be Apollo's mission making GraphQL effortless in JavaScript applications. Whereas GraphQL is only the query language which has its reference implementation in JavaScript, Apollo builds its ecosystem on top of it to make GraphQL available in a simpler way for a broader audience. This includes the client-side as well as the server-side, because they provide a large ecosystem of libraries for both sides. They even provide you with an intermediate layer with Apollo Engine as GraphQL gateway. So basically if you are interested in using GraphQL in your JavaScript applications, Apollo with its popular ecosystem is a great choice.
+GraphQL is only the query language that has a reference implementation in JavaScript, and Apollo builds its ecosystem on top to make GraphQL available for a wider audience. This includes the client-side as well as the server-side, because they provide a large ecosystem of libraries for both. The libraries provide an intermediate layer, with the Apollo Engine as a GraphQL gateway. Essentially there's a reason Apollo is one of the most popular choices for using GraphQL in JavaScript applications
 
 {{% chapter_header "Apollo Advantages" "apollo-advantages" %}}
 
-The following topics show you the major advantages of using Apollo.
+Major advantages of using Apollo.
 
 {{% sub_chapter_header "Apollo's Ecosystem" "apollo-ecosystem" %}}
 
-The Apollo ecosystem offers plenty of solutions for many problems even though GraphQL is in its early stages. And the ecosystem is growing. At every other conference the company is announcing an update for Apollo or another library which can be used complementary to Apollo's tech stack. For instance, that's why Apollo is not only covering GraphQL but also REST interfaces for enabling backward compatibility to RESTful architectures. It goes beyond GraphQL being used only for the network layer and remote data by offering a state management solution for local data too. So if you make the leap to use Apollo for your application, you are welcomed with a great ecosystem.
+While GraphQL is in its early stages, the Apollo ecosystem offers solutions for many of its challenges.  Beyond that we can see how much the ecosystem is growing, because the company announces an update for Apollo or another library that can be used with Apollo's tech stack at every other technology conference.  Apollo isn't just covering GraphQL, though; they also have effort invested in REST interfaces for backward compatibility to RESTful architectures.  This takes GraphQL beyond the network layer and remote data by offering a state management solution for local data, too. 
```

Comment:
"This takes GraphQL beyond the network layer" -> "They even take GraphQL beyond the network layer"